### PR TITLE
feat(button_component): create button component

### PR DESCRIPTION
**Pull request by mthanh2209**

### CODE REVIEW

File: src/assets/css/components/button.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  border: 1px solid #bcd0e5;
+  text-align: center;
+}
+.btn--primary {
+  padding: 10px 23px 9px 25px;
+  text-decoration: none;
+  flex-shrink: 0;
+  border-radius: 4px;
+  background: #02897a;
+}
```
Comment (hungnguyenhuu-agilityio) [inline src/assets/css/components/button.css:19]: I saw the above code from primary and secondary got the duplicate properties like padding, text-decoration, flex-shrink , border-radius. Should we merge them into one ?

Comment (mthanh2209) [reply src/assets/css/components/button.css:19]: Yes, We can merge the common properties of the .btn--primary and .btn--secondary classes

File: src/assets/css/components/button.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
+.btn {
+  color: var(--primary-color);
+  font-family: var(--primary-font);
+  font-size: 16px;
+  font-weight: var(--fw-semibold);
+  padding: 10px 25px 10px 25px;
```
Comment (hungnguyenhuu-agilityio) [inline src/assets/css/components/button.css]: Use short-hand please

Comment (mthanh2209) [reply src/assets/css/components/button.css]: sorry, i fixed it to padding: 10px 25px;

**Pull request merged**